docs/cloud/pouta/tutorials/image-pipeline.md

```diff
@@ -0,0 +1,650 @@
+
+We go back to `terminal_C`.
+We acquire a new address issuing the command:
+```
+$ openstack floating ip create public
+```
+
+The output will be similar to the following:
+```
++---------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Field               | Value                                                                                                                                                                                                |
++---------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| created_at          | 2025-02-04T13:13:38Z                                                                                                                                                                                 |
+| description         |                                                                                                                                                                                                      |
+| dns_domain          | None                                                                                                                                                                                                 |
+| dns_name            | None                                                                                                                                                                                                 |
+| fixed_ip_address    | None                                                                                                                                                                                                 |
+| floating_ip_address | 1.2.3.4                                                                                                                                                                                              |
+| floating_network_id | 26f9344a-2e81-4ef5-a018-7d20cff891ee                                                                                                                                                                 |
+| id                  | 1a38ae4f-1354-4958-b2be-72502b53c492                                                                                                                                                                 |
+| location            | Munch({'cloud': '', 'region_name': 'regionOne', 'zone': None, 'project': Munch({'id': 'ef20bcc4215d49ddbf33e5c5740d89fe', 'name': 'project_2003103', 'domain_id': 'default', 'domain_name': None})}) |
```

Comment:
Maybe mask also the project number

Reply:
Fixed in 20d01de112a6d165e495760b90848a3772ad32c0